lib/commands.sh

```
@@ -15,43 +108,135 @@ cmd_write() {
   require_allowed_author
   require_allowed_signer
 
+  local journal_ref="$(ref_journal "$env")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)